docs/reference/active_face_recognition.md

```diff
@@ -0,0 +1,199 @@
+# active_face_recognition module
+
+The *active_face_recognition* module contains the *ActiveFaceRecognitionLearner* class, which inherits from the abstract 
+class *LearnerRL*.
+
+### Class ActiveFaceRecognitionLearner
+Bases: `engine.learners.LearnerRL`
+
+The *ActiveFaceRecognitionLearner* is an agent that can be used to train a quadrotor robot equipped with an RGB camera 
+to maximize the confidence of OpenDR's FaceRecognition module.
+
+The [ActiveFaceRecognitionLearner](../../src/opendr/perception/active_perception/active_face_recognition/active_face_recognition_learner.py) class has the 
+following public methods:
+
+#### `ActiveFaceRecognitionLearner` constructor
+
+Constructor parameters:
+- **lr**: *float, default=3e-4*\
+  Specifies the initial learning rate to be used during training.
+- **iters**: *int, default=5e6*\
+  Specifies the number of steps the training should run for.
+- **batch_size**: *int, default=64*\
+  Specifies the batch size during training.
+- **checkpoint_after_iter**: *int, default=0*\
+  Specifies per how many training steps a checkpoint should be saved.
+- **checkpoint_load_iter**: *int, default=0*\
+  Specifies which checkpoint to load.
+- **temp_path**: *str, default=''*\
+  Specifies a path where the algorithm stores log files and saves checkpoints.
+- **device**: *{'cpu', 'cuda'}, default='cuda'*\
+  Specifies the device to be used.
+- **n_steps**: *int, default=6400*\
+  Specifies the number of steps to run environment per update.
+- **gamma**: *float, default=0.9*\
+  Discount factor for future rewards.
+- **clip_range**: *float, default=0.1*\
+  Clip policy updates.
+- **target_kl**: *float, default=0.1*\
+  KL Divergence update threshold.
```

Comment:
For consistency, we should either remove the "Specifies the" for each parameter description or add it to the last 3 descriptions. I would suggest to remove it to make it more compact.

Reply:
Even though we faced some issues with 2023b (scaled up objects on existing worlds), we do indeed use the latest version in this work as stated in the README.md (running the environment)

Reply:
In that case, we should probably fix the proto files and wbt files as they refer to Webots R2022b or R2023a.